eng/tools/tasks/Microsoft.DotNet.UnifiedBuild.MSBuildSdkResolver/Microsoft.DotNet.UnifiedBuild.MSBuildSdkResolver.csproj

```diff
@@ -1,7 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <!-- Use a NetStandard TFM bundled in the SDK to avoid prebuilts in source builds. -->
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
This change is needed to support SB.  netstandard2.0 is not bundled with the SDK.  It is included as a SBRP target pack.  I would like to eliminate the SBRP reference package from PSB as it eliminates special logic to handle them and increases the PSB size.  NS2.1 is bundled with the SDK so upgrading solves these problems.

Reply:
I think NS20 was chosen so that the msbuild sdk resolver works on both .NETCoreApp msbuild and desktop (VS) msbuild. If you upgrade beyond netstandard2.0 and with that lock out VS msbuild, you could also just go all up to NetCurrent.

Reply:
It looks like NS20 is required for at least winforms because the resolver is loaded into the repo level msbuild process.  We can use NetCurrent for non-windows to support SB scenarios and NS20 on windows.

Reply:
Any repo that uses desktop msbuild. aspnetcore falls into that bucket too.